src/controller/securemgr/verifier/verifier.go

```diff
@@ -245,48 +237,48 @@ func printAllHashFromContainerWhiteList() {
 	}
 }
 
```

Comment:
Currently the JWT verification is being done for add, delete, deleteall and print hash functions. So the verification is basically with in the same system and when user tries to add the hash. How about adding similar security when System A tries to access System B for offloading.

Reply:
As I wrote to Suresh, basic authentication for the external api is implemeted. I will do internal api later (I added internal requests on the router to exceptions, it means that we don't check internal requests).